internal/controller/driver_controller.go

```diff
@@ -504,6 +568,9 @@ func (r *driverReconcile) reconcileControllerPluginDeployment() error {
 											utils.CsiAddonsEndpointContainerArg,
 											"",
 										),
+										utils.If(logRotationSpec != nil, utils.LogToStdErrContainerArg, ""),
+										utils.If(logRotationSpec != nil, utils.AlsoLogToStdErrContainerArg, ""),
+										utils.If(logRotationSpec != nil, utils.LogFileContainerArg(fmt.Sprintf("%s-csi-%splugin", r.driver.Name, r.driverType)), ""),
```

Comment:
You need to pass the container name as well

Reply:
it is actually a `drivername-containername` `%s-(csi-%splugin)`

Reply:
Renamed